### PR TITLE
Ecosystem/Update Bitnami KB Stacks: wildfly, alfresco, tomcat, dokuwiki, drupal, prestashop, abantecart, django, jboss, diaspora, joomla, owncloud, elasticsearch, phpbb, jenkins, solr, mean

### PR DESCRIPTION
**Pull request by hk1232**

### CODE REVIEW

File: Ecosystem Partners/getting-started-with-alfresco-blueprint.md [hunk 1]
```diff
@@ -90,14 +87,12 @@ Alfresco Community is available as a Blueprint Package for deployment on an exis
 
   See the [Creating a new enterprise cloud server](../../servers/creating-a-new-enterprise-cloud-server/) KB for more information on completing this step.
 
-
 2. ** Select to Execute the Package on a Server Group**
 
-  Packages can be executed on one more more servers in a group.  Search for the public script package named **Install Alfresco Community on linux_TITLE**.
+  Packages can be executed on one more more servers in a group.  Search for the public script package named **Install Alfresco Community on Linux**.
 
   See the [using group tasks to install scripts on groups](../../servers/using-group-tasks-to-install-software-and-run-scripts-on-groups/) KB for more information on how to complete the next few steps.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-alfresco-blueprint.md]: broken link

File: Ecosystem Partners/getting-started-with-alfresco-blueprint.md [hunk 2]
```diff
@@ -90,14 +87,12 @@ Alfresco Community is available as a Blueprint Package for deployment on an exis
 
   See the [Creating a new enterprise cloud server](../../servers/creating-a-new-enterprise-cloud-server/) KB for more information on completing this step.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-alfresco-blueprint.md]: broken link

File: Ecosystem Partners/getting-started-with-diaspora-blueprint.md [hunk 3]
```diff
@@ -87,14 +85,12 @@ Diaspora is available as a Blueprint Package for deployment on an existing serve
 
   See the [Creating a new enterprise cloud server](../../servers/creating-a-new-enterprise-cloud-server/) KB for more information on completing this step.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-diaspora-blueprint.md]: broken link

File: Ecosystem Partners/getting-started-with-diaspora-blueprint.md [hunk 4]
```diff
@@ -87,14 +85,12 @@ Diaspora is available as a Blueprint Package for deployment on an existing serve
...
 
-
 2. ** Select to Execute the Package on a Server Group**
 
-  Packages can be executed on one more more servers in a group.  Search for the public script package named **Install Diaspora on linux_TITLE**.
+  Packages can be executed on one more more servers in a group.  Search for the public script package named **Install Diaspora on Linux**.
 
   See the [using group tasks to install scripts on groups](../../servers/using-group-tasks-to-install-software-and-run-scripts-on-groups/) KB for more information on how to complete the next few steps.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-diaspora-blueprint.md]: broken link

File: Ecosystem Partners/getting-started-with-drupal-blueprint.md [hunk 5]
```diff
@@ -11,7 +11,7 @@
 ### Description
 
 <img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/drupal/img/drupal-stack-110x117-9083fe6b191b13f79863eff74d3ce382.png" style="border:0;float:right;max-width:250px">
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-drupal-blueprint.md]: bad image link

**Comment by nathanyoung** [comment]
lots of broken links and images.

File: Ecosystem Partners/getting-started-with-drupal-blueprint.md (hunk 5 above)
Comment (hk1232) [inline Ecosystem Partners/getting-started-with-drupal-blueprint.md]: That image link works for me.  Can you reconfirm the issue?  Or is there a translation the tooling requires the protocol in the URL?

File: Ecosystem Partners/getting-started-with-jboss-blueprint.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+  "author": "<a href='https://twitter.com/KeithResar'>@KeithResar</a>",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+
+### Description
+
+<img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/jboss/img/jboss-stack-110x117-12361d85ff78442ab801caef8baf4d63.png" style="border:0;float:right;max-width:250px">
```
Comment (osbornm) [inline Ecosystem Partners/getting-started-with-jboss-blueprint.md]: Why the inline styles? Is our default style for images not correct?

Comment (osbornm) [inline Ecosystem Partners/getting-started-with-jboss-blueprint.md]: Can we also generate `alt` attributes for images?

Comment (hk1232) [inline Ecosystem Partners/getting-started-with-jboss-blueprint.md]: This is to make the logo images of varying sizes display consistently alongside the text.

Compare to images that don’t have this inline treatment:
https://www.centurylinkcloud.com/knowledge-base/ecosystem-partners/getting-started-with-apex-datacom-call-me-essentials-edition-blueprint/
https://www.centurylinkcloud.com/knowledge-base/ecosystem-partners/getting-started-with-alert-logic-web-security-manager-partner-template/

Comment (hk1232) [inline Ecosystem Partners/getting-started-with-jboss-blueprint.md]: What do you want it to contain?

Comment (osbornm) [inline Ecosystem Partners/getting-started-with-jboss-blueprint.md]: the alt attribute should contain a brief description of that the Image is displaying. For example the image tag displaying the CLC logo would have and alt attribute of 'CLC logo' where as a screenshot displaying something would have "Screenshot displaying feature XYZ"

**Pull request closed without merging**